### PR TITLE
update install instruction

### PR DESCRIPTION
**Pull request by MeWu-IDM**
@rhanIDM needs to install newer version to get fixes but we only provide installation instruction for releases. Propose this update, feel free to modify the description as appropriate.

### CODE REVIEW

File: pacehrh/vignettes/pacehrh.rmd [hunk 1]
```diff
@@ -246,11 +246,12 @@ The PACE-HRH model simulates healthcare needs in a given population to enable
 better health system staffing decisions. The model is configured via Excel
 sheets and run via an R script. 
 
+### Install using stable version binaries (windows) 
 1.  Close "R Model Inputs.xslx" in Excel. RStudio will be unable to load the input values
```
Comment (celiot-IDM) [inline pacehrh/vignettes/pacehrh.rmd:250]: File name needs to change.

File: pacehrh/vignettes/pacehrh.rmd [hunk 2]
```diff
@@ -260,3 +261,10 @@ sheets and run via an R script.
 
     The results are saved out to a CSV file with the date and name of the geography as specified
     in Excel in the filename. The console will beep at you when the model has finished running. 
+    
+### Install latest version via Github
```
Comment (celiot-IDM) [inline pacehrh/vignettes/pacehrh.rmd:265]: I think we need a different title for this section, something that makes it clear that by "latest" you mean "bleeding edge, not officially released" code.

File: pacehrh/vignettes/pacehrh.rmd [hunk 3]
```diff
@@ -260,3 +261,10 @@ sheets and run via an R script.
...
+
+Alternatively, one can install from repository source code with desired git reference which could be a commit, tag, or branch name
```
Comment (celiot-IDM) [inline pacehrh/vignettes/pacehrh.rmd:267]: 2 suggestions:
- Move this content to after the plain-vanilla example
- Include examples of installing from a commit and installing from a tag

I would, however, not be unhappy if we just omitted this content and focused entirely on the "tip of main" install.

**Review by celiot-IDM — COMMENTED** [review]
Comments included

File: pacehrh/vignettes/pacehrh.rmd (hunk 1 above)
Comment (JSchripsema-IDM) [reply pacehrh/vignettes/pacehrh.rmd:250]: Please also capitalize "Windows"

File: pacehrh/vignettes/pacehrh.rmd (hunk 3 above)
Comment (JSchripsema-IDM) [reply pacehrh/vignettes/pacehrh.rmd:267]: I agree with Charles and would add if we want to keep this, we give some indication of why you might want to install this way. I assume this is primarily for people who want to modify the code and verify behavior after doing so. Minor copyedits:

* Use "GitHub" capitalization
* "with the desired git reference, which" --note added word and comma
* Add a period at the end of the sentence

**Review by JSchripsema-IDM — APPROVED** [review]
Looks great, thank you! The comma added to the dev version instructions is misplaced though. It should read as follows:

Alternatively, one can install from repository source code with the desired git reference, which could be a commit, tag, or branch name.

**Review by celiot-IDM — COMMENTED** [review]
I have a couple of high-level concerns with the doc ...

1. Why do we start with the long description of how the spreadsheet works instead of a short "here's how to get something working right away" treatment?
2. The first para of the "Install and run the model" section is almost identical with the first paragraph of the doc ... which makes sense given that the "Install and run the model" section was originally the first section.
3. Step 1 ("Download the latest binary zip file") could do with being a bit more specific.
4. Steps 2 and 3 seem out of place ... they should happen *after* the package is installed, as described in steps 4, 5 , 6, and 7.
5. I think it would make more sense for Install and Run content to be in separate sections.

**Comment by JSchripsema-IDM** [comment]
@celiot-IDM 

I think the research workflow changed a bit after we moved to packaging releases via GitHub instead of including multiple binaries directly in the repo folder. It's hard to provide these kinds of structural edits when I can't edit the files directly--can we merge this PR and then I can make the suggested changes? 

1. "Press the button and run" seemed not particularly interesting from a research perspective. But we could move the info about how to run the model using defaults to the front. 
2. Definitely plagiarized myself.
3. What additional info would you like here? The releases page should provide an overview of the changes in each version.
4. Now that we've moved to the releases page on GitHub, I agree. The original order followed Brittany's workflow as I think she always (re)installed the latest binary when running the model. Now that there are multiple installation options and they all require you to go up the GH to fetch the binary, the order is off.
5. I can do this. Maybe reorder as: install, run, configure.

**Comment by MeWu-IDM** [comment]
@celiot-IDM @JSchripsema-IDM I am attending virtual test conference this week so may not get to this. Feel free to make updates. this PR is only intend to add the installation instruction so I did not change the existing steps

**Comment by celiot-IDM** [comment]
Let's merge-and-continue.

**Comment by celiot-IDM** [comment]
> @celiot-IDM
> 
> I think the research workflow changed a bit after we moved to packaging releases via GitHub instead of including multiple binaries directly in the repo folder. It's hard to provide these kinds of structural edits when I can't edit the files directly--can we merge this PR and then I can make the suggested changes?
> 
> 1. "Press the button and run" seemed not particularly interesting from a research perspective. But we could move the info about how to run the model using defaults to the front.
> 2. Definitely plagiarized myself.
> 3. What additional info would you like here? The releases page should provide an overview of the changes in each version.
> 4. Now that we've moved to the releases page on GitHub, I agree. The original order followed Brittany's workflow as I think she always (re)installed the latest binary when running the model. Now that there are multiple installation options and they all require you to go up the GH to fetch the binary, the order is off.
> 5. I can do this. Maybe reorder as: install, run, configure.

Thanks for the comments @JSchripsema-IDM !

Re #1: The point is to give people a quick way to know everything's working. And for those who want to reproduce and interrogate the original results before tweaking for themselves, it's exactly what they want. But I agree that they'll need the config instructions either soon or very soon.

Re #3: I was thinking maybe a screenshot of the GitHub releases area with an explanation that we mean the first file:

image


Re #5: LOVE IT!

**Pull request merged**